spark_expectations/notifications/plugins/email.py

```diff
@@ -50,7 +50,9 @@ def send_notification(
                 )
                 server.starttls()
                 text = msg.as_string()
-                server.sendmail(_context.get_mail_from, _context.get_to_mail, text)
+                server.sendmail(
+                    _context.get_mail_from, _context.get_to_mail.split(","), text
```

Comment:
can you also do strip() after split for each email, so that we don't have spaces.

Reply:
made the suggested changes...

Reply:
Thank you!